### DISCUSSION TITLE
Using bp3 theme in v5

### DISCUSSION THREAD

**Discussion by CptMaumau** [palantir/blueprint#6549] [Q&A] [answered]
Hello, is it possible to use bp3 namespace css after upgrading to v5? I wish to keep the old theme.

**Comment by adidahiya** [comment, marked as answer]
Nope, sorry. This unsupported out of the box. We'd prefer if you kept up with the visual design language changes in v4 and v5.

You can try to import Blueprint's Sass source files instead of its CSS and customize the default variable values to create a custom theme, but there are some caveats you must consider (see #6051).

Why do you wish to keep the visual design of Blueprint v3?
